archetypes/glossary.md

```diff
@@ -12,4 +16,4 @@ related_terms:
     - related_term1
     - related_term2
 ---
-Add glossary term definition and markdown here.
\ No newline at end of file
+Add glossary term definition and markdown here. For more information, <a href="example/with/link">see the documentation</a>.
```

Comment:
I think this is what is breaking the link checker. May just want to describe that you need to use an HTML link (or change it to link to something real, like https://www.w3schools.com/html/html_links.asp ?)

Reply:
Ah how embarrassing :)

Reply:
```suggestion
Add glossary term definition and markdown here. 
```